core-test/src/test/scala/org/bitcoins/core/script/ScriptProgramFactoryTest.scala

```diff
@@ -66,7 +66,15 @@ class ScriptProgramFactoryTest extends BitcoinSUnitTest {
       ScriptFlagFactory.empty)
     val program = PreExecutionScriptProgram(t)
     val inProgress =
-      ExecutionInProgressScriptProgram(t, stack, script, Nil, Nil, Nil, None)
+      ExecutionInProgressScriptProgram(t,
```

Comment:
Let's start using named parameters when we touch this now

```scala
                                       Nil,
                                       Nil,
                                       Nil,
                                       None,
                                       0,
                                       0
```

is not very descriptive

Reply:
I guess this is like the only place you didn't do it. So basically this is just a nit